R/parameters.R

```diff
@@ -382,7 +383,34 @@ generate_parameters_new_hospital_model <- function(
                                c(N_age_bins, N_trans_classes))
   trans_increase_array <- array(unlist(lapply(trans_increase, rep, N_age_bins)), 
                                 c(N_age_bins, N_trans_classes))
-  
+
+  #hospital_fitted is set to true, this will pull the fitted value from fitted_parameters.rds 
+  #this is calculated externally at the moment
+    if (hospital_fitted){
+    
+    fitted_parameter_file <- sircovid_file("extdata/fitted_parameters.rds")
+    
+    # Set up severity file into table
+    
+    
+    # Read file with fitted parameters (from Bob Verity's hospital model)
+    fitted_parameters <- readRDS(file = fitted_parameter_file)
```

Comment:
I'd prefer if we made this into a CSV file and read it in like we do with the severity data. If we end up merging #50 first this will need to be changed a bit too, as these go in the models rather than the parameters